portfolio/src/components/Technologies-list.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+const ListContainer = Styled.ul`
+    display: flex;
+    padding: 0;
+    max-width: 100%;
+    flex-wrap: wrap;
+    list-style-type: none;
+    color: rgba(255,255,255,0.7);
+    li :not(:last-child) {
+        padding-right: 6px;
+        margin-right: 6px;
+        border-right: 1px solid grey;
+    }
+`;
+const TechnologyLabel = Styled.li`
+    width: fit-content;
+`;
+
+const TechnologiesList = ({
+  technologies,
+}: {
+  technologies: Array<string>;
+}) => {
+  return (
+    <ListContainer>
+      {technologies.map((label) => (
```

Comment:
.map?

Reply:
.map is something built into javascript. It replaces the need to create a for loop and add items to a new array like this:

```js
  const names = ["gage", "camryn"];
  let namesCapitalized = [];

  for ( i = 0; i < names.length; i++) {
    namesCapitalized.push( names[i].toUpperCase() );
  }
```

.map just makes this less lines of code:
```js
  const names = ["gage", "camryn"];
  const namesCapitalized = names.map(name => name.toUpperCase());
```